cdap-system-app-api/src/main/java/io/cdap/cdap/api/service/worker/RunnableTaskContext.java

```diff
@@ -43,4 +47,40 @@ public void writeResult(byte[] data) throws IOException {
   public String getParam() {
     return param;
   }
+
+  @Nullable
+  public ClassLoader getArtifactClassLoader() {
```

Comment:
Unused method?

Reply:
Yes, this will be used in the follow up PR for pipeline validation task.